librz/include/rz_util/rz_str_util.h

```diff
@@ -6,6 +6,17 @@
 #define IS_SEPARATOR(x) ((x) == ' ' || (x) == '\t' || (x) == '\n' || (x) == '\r' || (x) == ' ' || \
 	(x) == ',' || (x) == ';' || (x) == ':' || (x) == '[' || (x) == ']' || \
 	(x) == '(' || (x) == ')' || (x) == '{' || (x) == '}')
+#define IS_COMPOUND_OP(x, y) \
+	(((x) == '+' && (y) == '+') || \
+		((x) == '-' && (y) == '-') || \
+		((x) == '+' && (y) == '=') || \
+		((x) == '-' && (y) == '=') || \
+		((x) == '*' && (y) == '=') || \
+		((x) == '/' && (y) == '=') || \
+		((x) == '%' && (y) == '=') || \
+		((x) == '&' && (y) == '=') || \
+		((x) == '|' && (y) == '=') || \
+		((x) == '^' && (y) == '='))
```

Comment:
i'm not sure if this is the correct location. i would define this in the rop headers? maybe call it RZ_ROP_IS_COMPOUND_OP

Reply:
Agree, it also could be an inline function instead.

Reply:
that is a good idea. an inline function is much better but do not add an inline function in the headers.

Reply:
Agreed :+1: , I have updated it as a `static inline` function